timed/settings.py

```diff
@@ -207,34 +207,34 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 
 OIDC_DEFAULT_BASE_URL = "http://timed.local/auth/realms/timed/protocol/openid-connect"
```

Comment:
Why is this not configurable?

Reply:
Good point!
It was initially only used for the dev setup, but definitely makes sense to expose this to have sane defaults for the userinfo and introspection endpoint.

Reply:
fixed